### PR TITLE
fix(datapoints-graph) Bumped package versions in order to fix missing asset selector icons.

### PR DESCRIPTION
**Pull request by eniosultan**
## Issue
The data points graph 2.0 was using different instance of ngx-components which caused issues with the asset selector. The issue was fixed at some point and in order to fix it here we need to bump the version of the package.

## Proposed changes
Bump @c8y/ngx-components, @c8y/cli, @c8y/styles @c8y/client versions

## Types of changes
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [X] I have read the [CONTRIBUTING](https://github.com/SoftwareAG/cumulocity-community-plugins/blob/main/CONTRIBUTING.md) doc
- [X] I have signed the [CLA](https://github.com/SoftwareAG/cumulocity-community-plugins/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cumulocity-community-plugins",
-  "version": "2.1.5",
+  "version": "2.1.6",
```
Comment (jdreSAG) [inline package.json:3]: version will be bumped automatically, revert this change

**Review by jdreSAG — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (reey) [inline package-lock.json:1]: there are way to many changes in this file from my point of view..

File: package.json
Comment (reey) [inline package.json:1]: `@c8y/cli` was missed..

Comment (eniosultan) [reply package.json:1]: added

File: package-lock.json
Comment (eniosultan) [reply package-lock.json:1]: reverted

**Comment by eniosultan** [comment]
> please also add a little bit more description in PR description- what was exact issue, what was causing it (not sharing ngx-components between plugin and shell). In 1 month we will forget what was the issue and this repo has no link with our internal JIRA

done

**Review by janhommes — APPROVED** [review]

**Pull request merged**